schema.json

```diff
@@ -1462,6 +1462,587 @@
     ]
   },
   {
+    "name": "Optimized Checkout",

```

Comment:
Yeah it could replace `this.formatId(name) === 'optimizedcheckout'`
Would it be much cleaner (and worth changing the theme `schema.json`), rather than just checking if `"name": "Optimized Checkout"` exists in the theme ?